build/build.xml

```diff
@@ -101,6 +108,17 @@ External Dependencies:
 		<property name="src.isInit" value="true" />
 	</target>
 
+	<!-- create script to run built commandbox using sources (for core cfml development) -->

```

Comment:
You mean you symlink the cfml directory from your "normal" ${user.home}/.CommandBox to the git dir?

This is a little safer than a symlink, as it also uses a temporary commandbox home that is deleted whenever you build.  Just gives me a bit more peace of mind, and it's sorta-kinda integration testing at the same time.

I pretty often wipe out my ~/.CommandBox home, but the idea is that this'll help other folks to contribute, without fear of wiping out their servers and stuff or existing artifacts/whatnot.  Since the dist directory is different with each version, it needed to be in the build vs. just a file in the build directory.  The bonus is all I have to remember is build.runsrc, vs. parameters or if the target comes first or last with a symlink. ;-)